config/chime_science_run_recv_21cm.yaml

```diff
@@ -0,0 +1,63 @@
+##########################################
+#
+# chime_science_run_recv_21cm.yaml
+#
+# CHIME 21cm absorber receiver node configuration used in the December 2019 run.
```

Comment:
I would remove the date reference, and include some details about how it writes out the files (i.e. raw output).